presentation/src/main/java/com/stop/model/Event.kt

```diff
@@ -0,0 +1,19 @@
+package com.stop.model
+
+open class Event<out T>(private val content : T){
```

Comment:
```suggestion
class Event<out T>(private val content : T){
```
화면 회전에 따른 LiveData 핸들링 좋습니다👍
Event 클래스를 상속하는 클래스가 없기 때문에 open 접근자는 지워주는 것이 좋을 것 같습니다!

Reply:
0c4a797c7eee500921191d918d2b27f425450ec6